content/streamlit-cloud/deploy-your-app/index.md

```diff
@@ -55,15 +55,15 @@ Although you can deploy multiple apps from the same repository, there can be onl
 
 ## Advanced settings for deployment
 
-3. (Optional) If you are connecting to a data source or want to specify the Python version for your app, you can do that by clicking "**Advanced settings**" before you deploy the app. Learn more about [Secrets management](/streamlit-community-cloud/deploy-your-app/secrets-management).
+<Note>
 
-   ![Advanced settings for deploying your app](/images/streamlit-community-cloud/deploy-an-app-advanced.png)
+Streamlit Community Cloud supports all released [versions of Python](https://devguide.python.org/versions/) that are still receiving security updates. Streamlit Community Cloud defaults to version 3.9. You can select a version of your choice from the "Python version" dropdown in the "Advanced settings" modal. If an app is running a version of Python that becomes unsupported, it will be forcibly upgraded to oldest, supported version of Python and may break.
```

Comment:
> If an app is running a version of Python that becomes unsupported, it will be forcibly upgraded to oldest, supported version of Python and may break.

Confirm is this actually true? From what I recall talking with the Cloud engineers when we deprecated 3.7 support, we just removed the ability to deploy new apps to 3.7 but didn't force upgrade old apps (at least not all at once).

Reply:
This came from Brian Holt. (His Slack comment is pasted into the Notion task.)